cdap-docs/reference-manual/source/http-restful-api/http-restful-api-v3/metrics.rst

```diff
@@ -308,39 +121,27 @@ Flowlet, Worker, Procedure, Mapper, or Reducer level:
    * - Dataset Metric
      - Context
    * - A single Dataset in the context of a single Flowlet
-     - ``/datasets/<dataset-id>/apps/<app-id>/flows/``
-       ``<flow-id>/flowlets/<flowlet-id>``
+     - ``namespace.<namespace>.datasets.<dataset-id>.apps.<app-id>.flows.<flow-id>.flowlets.<flowlet-id>``
    * - A single Dataset in the context of a single Flow
-     - ``/datasets/<dataset-id>/apps/<app-id>/flows/<flow-id>``
+     - ``namespace.<namespace>.datasets.<dataset-id>.apps.<app-id>.flows.<flow-id>``
    * - A single Dataset in the context of a specific Application
-     - ``/datasets/<dataset-id>/<any application context>``
+     - ``namespace.<namespace>.datasets.<dataset-id>.apps.<app-id>``
    * - A single Dataset across all Applications
-     - ``/datasets/<dataset-id>``
+     - ``namespace.<namespace>.datasets.<dataset-id>``
    * - All Datasets across all Applications
-     - ``/``
+     - ``namespace.<namespace>.datasets.*``
 
-Querying by Run-ID
-------------------
-
-Each execution of an program (Flow, MapReduce, Spark, Services, Worker, Procedure) has an associated run-ID that uniquely identifies that program's run.
-We can query metrics for an program by its run-ID to see the metrics for a particular run.
-Please see the :ref:`Run Records and Schedule <rest-program-runs>` on retrieving active and historical program runs.
-
-When querying by run-ID, it is specified after the ``program-id`` in the path::
-
-  /apps/<app-id>/<program-type>/<program-id>/runs/<run-id>/
+.. _available-metrics:
 
-Examples ::
-
-    GET /apps/<app-id>/flows/<flow-id>/runs/<run-id>/flowlets/<flowlet-id>/
-    GET /apps/<app-id>/mapreduce/<mapreduce-id>/runs/<run-id>/
+Available System Metrics
+------------------------
+For CDAP metrics (system metrics), the available metrics depend on the context.
+User-defined metrics are available in the context that they are emitted from.
 
-Available Metrics
------------------
-For CDAP metrics, the available metrics depend on the context.
-User-defined metrics will be available at whatever context that they are emitted from.
+Note that a user metric may have the same name as a system metric; they are distinguished 
```

Comment:
Note that user metric emitted in a program may have the same name as system metric; they are distinguished by prepending the respective prefix when querying.